aptos-move/mvhashmap/src/versioned_group_data.rs

```diff
@@ -289,14 +289,13 @@ impl<
         // reading an entry by a lower txn index. However, if the size has changed, then
         // those read dependencies will be added to invalidated_dependencies, and the
         // store_deps variable will be empty.
-        let mut store_deps = BTreeSet::new();
+        let mut store_deps = BTreeMap::new();
```

Comment:
In if below, we re-assign a map here on passed validation. Should we return `new_deps` on passed validation, and empty map otherwise?